postgres/init/90-public-efhistory.sql

```diff
@@ -57,7 +57,8 @@ COPY public.__efmigrations_history_portal (migration_id, product_version) FROM s
 20220909133703_CPLP-1330-NotificationAppReleaseRequest	6.0.4
 20220914063503_CPLP-1378-AddNewDocumentTypes	6.0.4
 20220914081150_CPLP-1406-AddAgreementAssignedApps	6.0.4
-20220916132329_CPLP-1406-AddConsentAssignedOfferSubscription	6.0.4
+20220915073657_CPLP-1409-ExpandNotificationTypes	6.0.4
```

Comment:
That migration apparently was created after 20220916132329_CPLP-1406-AddConsentAssignedOfferSubscription, it needs to be recreated based on latest main and re-applied. Please be aware that https://github.com/catenax-ng/product-portal-backend/pull/214 also contains an outstanding migration that is likely to be merged first. We therefore postpone this PR until said migration has been applied.